org.jacoco.core/src/org/jacoco/core/internal/analysis/filter/LombokGeneratedFilter.java

```diff
@@ -0,0 +1,45 @@
+/*******************************************************************************
+ * Copyright (c) 2009, 2017 Mountainminds GmbH & Co. KG and Contributors
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *    Rüdiger zu Dohna - initial API and implementation
+ *
+ *******************************************************************************/
+package org.jacoco.core.internal.analysis.filter;
+
+import java.util.List;
+
+import org.objectweb.asm.tree.AnnotationNode;
+import org.objectweb.asm.tree.MethodNode;
+
+/**
+ * Filter for methods generated by Lombok which are annotated with
+ * <code>@lombok.Generated</code>.
+ */
+public class LombokGeneratedFilter implements IFilter {
+
+	public void filter(final MethodNode methodNode,
+			final IFilterOutput output) {
+		if (hasLombokGeneratedAnnotation(methodNode)) {
+			output.ignore(methodNode.instructions.getFirst(),
+					methodNode.instructions.getLast());
+		}
+	}
+
+	private boolean hasLombokGeneratedAnnotation(final MethodNode methodNode) {
+		final List<AnnotationNode> list = methodNode.invisibleAnnotations;
```

Comment:
@marchof I'd prefer to name local variable `invisibleAnnotations` or even better `runtimeInvisibleAnnotations` instead of just `list`